libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/types.ts

```diff
@@ -0,0 +1,168 @@
+// (C) 2021 GoodData Corporation
+import { ObjRef } from "@gooddata/sdk-model";
+import isEmpty from "lodash/isEmpty";
+
+// Grantee types
+
+/**
+ * @internal
+ */
+export type GranteeItem = IGranteeUser | IGranteeGroup | IGranteeGroupAll;
+
+/**
+ * @internal
+ */
+export type GranteeType = "user" | "group" | "groupAll";
+
+/**
+ * @internal
+ */
+export interface IGranteeBase {
+    granteeType: GranteeType;
+    id: ObjRef;
+}
+
+/**
+ * @internal
+ */
+export interface IGranteeUser extends IGranteeBase {
+    granteeType: "user";
```

Comment:
I would just name these `type`, `name`, `email`, no need to prefix it as it can be guessed from the interface name.

Reply:
done